packages/sanes-chrome-extension/src/logic/persona/persona.ts

```diff
@@ -142,7 +151,8 @@ export class Persona {
 
   public startSigningServer(
     authorizeGetIdentities?: GetIdentitiesAuthorization,
-    authorizeSignAndPost?: SignAndPostAuthorization
+    authorizeSignAndPost?: SignAndPostAuthorization,
```

Comment:
Why those attrs are optional? I do not see another use case for the signing service without those callbacks, right?

Reply:
The only reason is unit testing. If you want we can make them non-optional and move the default implementations into the test file.